### ISSUE TITLE
[Enhancement] Device type: Vacuum

### ISSUE THREAD

**Issue by Lackmake** [t0bst4r/home-assistant-matter-hub#69]
Would be nice to have support for vacuum cleaners. 
(Valetudo refuses to support matter, but i'd like to start my vacuum with Google Assistant and i could bridge this gap with this :) )

**Comment by steuerlexi** [comment]
for the time being you could can set up a switch, script, automation as a helper to start your vacuum using this great integration

**Comment by nicoodeimos** [comment]
Would be amazing to have this!

**Comment by Casuallynoted** [comment]
Would also love to see this now that HomeKit has support in the iOS 18.3 beta.

**Comment by BorjaIglesias** [comment]
Looking for this feature!Hope Will come soon!Thanks!

**Comment by pr8x** [comment]
There seems to be some motion on this: https://github.com/t0bst4r/home-assistant-matter-hub/pull/386 🎉

**Comment by t0bst4r** [comment]
You are right. I started implementing this feature (#386), but found a bug in the matter.js framework (project-chip/matter.js#1612) which blocks us from finally implementing it.

**Comment by ovflowd** [comment]
@t0bst4r it seems like the upstream bug is fixed? https://github.com/project-chip/matter.js/issues/1612
